linux/cjis-prerequisites.yml

```diff
@@ -0,0 +1,22 @@
+---
+# The CJIS role seems to assume these packages are installed and the
```

Comment:
@l3acon i haven't tested this yet, but i took a look at the CJIS role and i believe you won't need to install and enable firewalld and NetworkManager if the following variables get set:

```
service_firewalld_enabled: false
firewalld_sshd_port_enabled: false
```

these can be added to the [LINUX / Multi-profile Compliance](https://github.com/ansible/product-demos/blob/10f0bb46414c56481a8fb0e58a4d927f2c9b2235/linux/setup.yml#L362) job template as long as the other CaC roles don't use variables of the same name.